sanssouci/lambda_calibration.py

```diff
@@ -87,54 +83,54 @@ def get_permuted_p_values(X, labels, B=100, row_test_fun=stats.ttest_ind):
     return pval0
 
 
-def get_permuted_p_values_one_sample(X, B=100, seed=None):
+def get_permuted_p_values_one_sample(X, B=100, seed=None, n_jobs=1):
     """
     Get permutation p-values: Get a matrix of p-values under the null
     hypothesis obtained by sign-flipping (one-sample test).
-
     Parameters
     ----------
-
     X : array-like of shape (n,p)
         numpy array of size [n,p], containing n observations of p variables
         (hypotheses)
     B : int
         number of sign-flippings to be performed (default=100)
-
+    n_jobs : int
+        number of CPUs used for computation. Default = 1
     Returns
     -------
-
     pval0 : array-like of shape (B, p)
         A numpy array of size [B,p], whose rows are sorted increasingly.
         The entry i,j corresponds to p_{(j)}(g_i.X) with notation of [1]
         (section 4.5)_
-
     References
     ----------
-
     .. [1] Blanchard, G., Neuvial, P., & Roquain, E. (2020). Post hoc
         confidence bounds on false positives using reference families.
         Annals of Statistics, 48(3), 1281-1303.
     """
 
     np.random.seed(seed)
-
-    # Init
+    seeds = np.random.randint(np.iinfo(np.int32).max, size=B)
```

Comment:
To allow a fine control of the method, why not have seed as kw argument ?
If not provided, the above line could be the default.

Reply:
Just making sure I understand: do you mean having the "seeds" vector of length B as a kw argument instead of the current arg "seed"?

Reply:
No. in L.113, seeds could just be taken from an input argument. This offers external control (otherwise people need to read the code to figure out how seed is set).

Reply:
```
rng = check_random_state(random_state)
seeds = rng.randint(np.iinfo(np.int32).max, size=B)```
where random_state is an optional argument to the function.

Reply:
thanks @bthirion